debian/control

```diff
@@ -14,6 +14,7 @@ Architecture: all
 Depends:
   ${misc:Depends},
   ${python3:Depends},
+  node-red-contrib-revpi-nodes (= 1.1.0+ds1-1+revpi11+1),
```

Comment:
```suggestion
  node-red-contrib-revpi-nodes (>= 1.1.0),
  node-red-contrib-revpi-nodes (<< 1.1.1)
```
What about this approach?

Reply:
Of course, in the current case, we have the problem that server and nodes always have to be released as new packages at the same time. But we also currently have patches in our Debian package to fix errors from the upstream without their release.

Of course, an automatic update would always be possible, but if a fix has to be forced over a new debian release version, we can't do that with the proposal above.

🤔